python/mage/node_classification/utils/extract_from_database.py

```diff
@@ -0,0 +1,262 @@
+import torch
+from torch_geometric.data import Data, HeteroData
+import mgp
+import numpy as np
+from tqdm import tqdm
+import random
+from collections import Counter
+import torch_geometric.transforms as T
+import typing
+
+def extract_from_database(
+    ctx: mgp.ProcCtx,
+    train_ratio: float,
+    features_name: str,
+    class_name: str,
+) -> typing.Tuple[HeteroData, str, typing.Dict, typing.Dict]:
+    
+    data = HeteroData()
+    
+
+    #################
+    # NODES
+    #################
+    nodes = list(iter(ctx.graph.vertices))
+    node_types = []
+    embedding_lengths = {}
+    observed_attribute = None
+    
+    for i in range(len(nodes)):
+        if type(nodes[i].properties.get(features_name)) == type(None):
+            continue
+        node_types.append(nodes[i].labels[0].name)
+        if nodes[i].labels[0].name not in embedding_lengths:
```

Comment:
Check if there is label in labels. Cover the case with multi-labels

Reply:
Discuss on meeting. What should be `nodes[i].labels` and how can I be a priori sure about it?